hi-ml-azure/src/health_azure/utils.py

```diff
@@ -89,6 +89,7 @@
     "RSLEX_DIRECT_VOLUME_MOUNT": "true",
     "RSLEX_DIRECT_VOLUME_MOUNT_MAX_CACHE_SIZE": "1",
     "DATASET_MOUNT_CACHE_SIZE": "1",
+    "AZUREML_COMPUTE_USE_COMMON_RUNTIME": "true"
```

Comment:
attempt to fix aml folder structure

Reply:
I am not sure it's completly solved :( this smoke test is[ still failing ](https://ml.azure.com/experiments/id/f1046b8c-4cb3-4abc-b637-ba27ee9475dc/runs/TilesPandaImageNetMIL_1659444085_fa70e6cc?wsid=/subscriptions/a85ceddd-892e-4637-ae4b-67d15ddf5f2b/resourceGroups/health-ml/providers/Microsoft.MachineLearningServices/workspaces/hi-ml&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#outputsAndLogs)
unable to delete the val_old folder 
<img width="527" alt="image" src="https://user-images.githubusercontent.com/37396332/182395978-8c3160d7-eccd-4e5b-87d5-f3a22d737025.png">

Because of this fuse_hidden file
<img width="281" alt="image" src="https://user-images.githubusercontent.com/37396332/182395819-3c64a13c-e8ae-4f66-b727-303fd69ed95d.png">
Any thoughts? 
Do you think it's safe to [ignore errors](https://docs.python.org/3/library/shutil.html#shutil.rmtree) here?

Reply:
I set for now ignore_errors=True

Reply:
Also, I had to set the AML runtime env var to False in the meantime.